docs/conf.py

```diff
@@ -295,3 +295,12 @@
 # ------------------------------------
 
 hoverxref_auto_ref = True
+hoverxref_project = "scrapy"
+hoverxref_version = release
```

Comment:
Hi! This breaks in production when rendered by Read the Docs.

You shouldn't override `hoverxref_version` and `hoverxref_project` since they are taken automatically from Read the Docs.

If you want to avoid your CI failing because of this, you can define the environment variables as Read the Docs does:

- READTHEDOCS_PROJECT=scrapy
- READTHEDOCS_VERSION=''

With the current configuration, _all the versions built on Read the Docs_ will point to a different _version_ on Read the Docs and this will conflict. For example, current `master` version in Read the Docs defines `hoverxref_version='2.0.0'` but that version does not exist on Read the Docs and the tooltip does not known where to get the content from.

Cheers,

Reply:
Thanks for the heads up @humitos!

Reply:
Many thanks for looking into this @humitos! :rocket: